web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/forvaltning/stonadsstatistikk/StønadsstatistikkMigreringTask.java

```diff
@@ -0,0 +1,122 @@
+import jakarta.inject.Inject;
+import jakarta.persistence.EntityManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import no.nav.foreldrepenger.behandling.BehandlingReferanse;
+import no.nav.foreldrepenger.behandlingslager.behandling.Behandling;
+import no.nav.foreldrepenger.behandlingslager.behandling.repository.BehandlingRepository;
+import no.nav.foreldrepenger.behandlingslager.behandling.vedtak.BehandlingVedtak;
+import no.nav.foreldrepenger.behandlingslager.behandling.vedtak.BehandlingVedtakRepository;
+import no.nav.foreldrepenger.behandlingslager.fagsak.FagsakProsesstaskRekkefølge;
+import no.nav.foreldrepenger.datavarehus.v2.StønadsstatistikkTjeneste;
+import no.nav.foreldrepenger.skjæringstidspunkt.SkjæringstidspunktTjeneste;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+
+@Dependent
+@ProsessTask(value = "stønadsstatisikk.migrering", maxFailedRuns = 1)
+@FagsakProsesstaskRekkefølge(gruppeSekvens = false)
+class StønadsstatistikkMigreringTask implements ProsessTaskHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(StønadsstatistikkMigreringTask.class);
+    static final String DATO_KEY = "dato";
+    static final String SISTE_DATO = "sistedato";
+    static final String SEKUNDER_PAUSE = "sekunderPause";
+
+    private final BehandlingRepository behandlingRepository;
+    private final EntityManager entityManager;
+    private final StønadsstatistikkTjeneste stønadsstatistikkTjeneste;
+    private final SkjæringstidspunktTjeneste skjæringstidspunktTjeneste;
+    private final BehandlingVedtakRepository behandlingVedtakRepository;
+    private final ProsessTaskTjeneste prosessTaskTjeneste;
+
+    @Inject
+    public StønadsstatistikkMigreringTask(BehandlingRepository behandlingRepository,
+                                          EntityManager entityManager,
+                                          StønadsstatistikkTjeneste stønadsstatistikkTjeneste,
+                                          SkjæringstidspunktTjeneste skjæringstidspunktTjeneste,
+                                          BehandlingVedtakRepository behandlingVedtakRepository,
+                                          ProsessTaskTjeneste prosessTaskTjeneste) {
+        this.behandlingRepository = behandlingRepository;
+        this.entityManager = entityManager;
+        this.stønadsstatistikkTjeneste = stønadsstatistikkTjeneste;
+        this.skjæringstidspunktTjeneste = skjæringstidspunktTjeneste;
+        this.behandlingVedtakRepository = behandlingVedtakRepository;
+        this.prosessTaskTjeneste = prosessTaskTjeneste;
+    }
+
+    @Override
+    public void doTask(ProsessTaskData prosessTaskData) {
+        var dato = LocalDate.parse(prosessTaskData.getPropertyValue(DATO_KEY), DateTimeFormatter.ISO_LOCAL_DATE);
```

Comment:
Hadde det vært fint å se at det kunne være flere datoer det skal migreres for her?